docs/fundamentals/dashboard/explore.md

```diff
@@ -1,52 +1,45 @@
 ---
-title: .NET Aspire dashboard overview
+title: Explore .NET Aspire dashboard
 description: Explore the .NET Aspire dashboard features through the .NET Aspire Starter app.
-ms.date: 03/13/2024
+ms.date: 04/23/2024
 ms.topic: reference
 ---
 
-# .NET Aspire dashboard overview
-
-[.NET Aspire](../get-started/aspire-overview.md) project templates offer a sophisticated dashboard for comprehensive app monitoring and inspection. This dashboard allows you to track closely various aspects of your application, including logs, traces, and environment configurations, in real-time. It's purpose-built to enhance the local development experience, providing an insightful overview of your app's state and structure.
+# Explore the .NET Aspire dashboard
 
 In the upcoming sections, you'll discover how to create a .NET Aspire app and embark on the following tasks:
 
 > [!div class="checklist"]
 >
-> - Investigate the dashboard's capabilities by using the app generated from the project template as explained in the [Quickstart: Build your first .NET Aspire app.](../get-started/build-your-first-aspire-app.md)
+> - Investigate the dashboard's capabilities by using the app generated from the project template as explained in the [Quickstart: Build your first .NET Aspire app.](../../get-started/build-your-first-aspire-app.md)
 >
 > - Delve into the features of the .NET Aspire dashboard app.
 
 The screenshots featured in this article showcase the dark theme. For more details on theme selection, refer to [Theme selection](#theme-selection).
 
-## Explore the .NET Aspire dashboard
+## Dashboard authentication
 
-The .NET Aspire dashboard is only visible while the **AppHost** app is running and will launch automatically when you start the project. The left navigation provides links to the different parts of the dashboard, each of which you'll explore in the following sections. Additionally, the cog icon in the upper right of the dashboard provides access to the settings page, which allows you to configure your dashboard experience.
+When you run a .NET Aspire app host, the orchestrator starts up all the apps dependent resources and then opens a browser window to the dashboard. The .NET Aspire dashboard requires token-based authentication for its users because it displays environment variables and other sensitive information.
```

Comment:
```suggestion
When you run a .NET Aspire app host, the orchestrator starts up all the app's dependent resources and then opens a browser window to the dashboard. The .NET Aspire dashboard requires token-based authentication for its users because it displays environment variables and other sensitive information.
```